test/helpers/hosts_and_hostgroups_helper_test.rb

```diff
@@ -187,9 +188,24 @@ def setup
     @hostgroup.content_source = @content_source
 
     @host = ::Host.new(:architecture => @arch, :operatingsystem => @os,
-                      :content_facet_attributes => {:lifecycle_environment_id => @env.id,
-                                                    :content_view_id => @cv.id,
-                                                    :content_source_id => @content_source.id})
+                       :content_facet_attributes => {:lifecycle_environment_id => @env.id,
+                                                     :content_view_id => @cv.id,
+                                                     :content_source_id => @content_source.id},
+                       :location => Location.new(
+                         name: 'Default Location',
+                         type: 'Location',
+                         ignore_types: ['ProvisioningTemplate', 'Hostgroup'],
+                         label: 'Default_Location',
+                         title: 'Default Location'
+                       ),
+                       :organization => Organization.new(
+                          name: 'Default Organization',
+                          type: 'Organization',
+                          label: 'Default_Organization',
+                          title: 'Default Organization'
+                       )
+
```

Comment:
nitpick: unnecessary blank line

Reply:
it will happen to me from time to time (trying not to do it though), I love empty spaces :D